src/govuk/template.njk

```diff
@@ -28,7 +28,7 @@
     <meta property="og:image" content="{{ assetUrl | default('/assets') }}/images/govuk-opengraph-image.png">
   </head>
   <body class="govuk-template__body {{ bodyClasses }}" {%- for attribute, value in bodyAttributes %} {{attribute}}="{{value}}"{% endfor %}>
-    <script>document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
+    <script{% if scriptNonce %} nonce="{{ scriptNonce }}"{% endif %}>document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
```

Comment:
That's correct @36degrees, a single cryptographically secure random token is generated server side and included in the `Content-Security-Policy`

Reply:
@36degrees yes, if we have more inline scripts they can use the same `nonce`.  I wouldn't personally call it `inlineScriptCspNonce` as it can be used on remote scripts too.  Adding CSP makes sense to me, should we go for `scriptCspNonce`?